pypit/traceslits.py

```diff
@@ -42,10 +46,10 @@
                                        'extrapolate': {'pos': 0, 'neg':0}},
                                'sobel': {'mode': 'nearest'}}})
 
-# TODO -- Add data model for MasterFrame here
+#  See save_master() for the data model for output
 
 
-class TraceSlits(object):
+class TraceSlits(masterframe.MasterFrame):
```

Comment:
Ah, maybe this is why you have multiple inheritance?  So you can treat TraceSlits as a MasterFrame, but TraceImage as a ProcessImages?

Reply:
exactly! :)